packages/enclave-sdk/src/enclave-sdk.ts

```
@@ -86,6 +109,64 @@ export class EnclaveSDK {
+    circuit: CompiledCircuit,
+  ): Promise<VerifiableEncryptionResult> {
+    await initializeWasm();
+    switch (this.protocol) {
+      case FheProtocol.BFV:
+        const [encryptedVote, circuitInputs] = bfv_verifiable_encrypt_number(
+          data, 
+          publicKey,
+          this.protocolParams.degree,
+          this.protocolParams.plaintextModulus,
+          this.protocolParams.moduli,
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)